src/containers/Chat/ChatMessages/ChatMessage/ChatMessage.tsx

```diff
@@ -139,7 +145,12 @@ export const ChatMessage = ({
     );
   }
 
-  const isSender = sender.id === contactId;
+  let isSender: boolean;
+  if (groups) {
+    isSender = status === 'received';
+  } else {
+    isSender = sender.id === contactId;
+  }
```

Comment:
Didn't understand this logic.

Reply:
For WhatsApp groups, there is no sender/receiver id so to determine is the message is sent or received, status(received/sent) is used